modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,35 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_option_table_threshold', 100 );
+	return $wpdb->get_results( $wpdb->prepare( "SELECT option_name, LENGTH(option_value) AS option_value_length FROM `wp_options` WHERE autoload='yes' AND LENGTH(option_value) > %s ORDER BY option_value_length DESC LIMIT 20", $option_threshold ) );
```

Comment:
`$option_threshold` value should be an integer so we should use `%d` for a better result otherwise anyone case uses a string as a filter result and gets the wrong result in a query.